go/oasis-net-runner/fixtures/default.go

```diff
@@ -35,6 +41,28 @@ var (
 	keymanagerID common.Namespace
 )
 
+// GetFixture generates a fixture object from given file or default fixture, if no fixtures file provided.
+func GetFixture() (f *oasis.NetworkFixture, err error) {
```

Comment:
This (together with new flags and the relevant init section) should probably not be in `fixtures/default.go` but rather in `fixtures/fixtures.go`. The file-based fixture should then be in `fixtures/file.go`.